Src/ReleaseHistory.md

```diff
@@ -16,6 +16,10 @@
 - UER => eliminate unhandled exceptions in rules
 - UEE => eliminate unhandled exceptions in engine
 
+## Unreleased
+
+- RE2.Native will compile with the latest Windows SDK 10.0.* installed. [#607](https://github.com/microsoft/sarif-pattern-matcher/pull/607)
```

Comment:
>Native [](http://example.com/codeflow?start=6&length=6)

We don't want this note. Customers should not receive an arbitrary Windows SDK, and we don't need to document a change that's relevant only to dev builds.